templates/default/sql/dml/1.3.0.sql.erb

```diff
@@ -11,3 +11,7 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ext_hostname';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ssl_hostname';
 REPLACE INTO `hopsworks`.`variables` VALUES ("hopsworks_enterprise", "<%= node['install']['enterprise']['install']%>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("kubernetes_installed", "<%= node['install']['kubernetes'] %>");
+DELETE FROM `hopsworks`.`variables` WHERE `id`='logstash_ip';
+DELETE FROM `hopsworks`.`variables` WHERE `id`='logstash_port';
+DELETE FROM `hopsworks`.`variables` WHERE `id`='logstash_port_tf_serving';
+DELETE FROM `hopsworks`.`variables` WHERE `id`='logstash_port_sklearn_serving';
```

Comment:
These are application specific ports, how will hopsworks discover the endpoint (IP:port) if we remove these ports from the `variables table` ?

Reply:
Through consul, however, i checked the code in hopsworks and the logstash ports and ips are not used at all.
https://github.com/logicalclocks/hopsworks-ee/pull/47/files

Reply:
Sent pm